.changeset/boolean-bananas-bounce.md

```diff
@@ -4,8 +4,7 @@
 
 Type inference is now able to handle logical expressions: `&&`, `||`, and `??`.
 
-
-**Examples**
+## Examples
```

Comment:
As explained here, we can't use big headers in changesets because they break the changelog and upstream tools

https://github.com/biomejs/biome/pull/6748

You can use other others, but as also explained in the PR. It doesn't make much sense to create a header (hence a link) to an example if there isn't something "bigger". 

For this case you'd want 

```
##### Feature 

###### Example
```

Otherwise, it's best to keep the `strong` format